build.gradle

```diff
@@ -354,35 +236,6 @@ task acceptanceTest(type: Test) {
     classpath = sourceSets.acceptanceTest.runtimeClasspath
 }
 
-def execAndWait(command, timeoutInSeconds = 15) {
-    println "Running command: " + command
-    ProcessBuilder pb = new ProcessBuilder(["bash", "-c", command]).inheritIO();
-    Process proc = pb.start();
-    proc.waitFor(timeoutInSeconds, TimeUnit.SECONDS);
-    return proc.exitValue()
-}
-
-task startDockerContainerAndWait() {
-    // start app with docker
-    dependsOn startDockerContainer
-
-    doLast {
-        // wait till application is up (health check is successful)
-        int result = 1
-        while (result != 0) {
-            result = execAndWait('curl http://localhost:8080/health')
-            sleep(1000L)
-        }
-    }
-    outputs.upToDateWhen { false }
-}
-
-task fullAcceptanceTest(type: GradleBuild) {
```

Comment:
what I miss now comparing to old build file:
1) we now don't have a single task that will run Nakadi with all its dependencies
2) we now don't have a single task that will run acceptance tests (together with Nakadi and all its dependencies)

So I think docker-compose should be integrated to gradle build.